### ISSUE TITLE
3.6.4rc8 osgconv doesn't work for .obj

### ISSUE THREAD

**Issue by artoonie** [openscenegraph/OpenSceneGraph#800]
Reverting back to 3.6.3 does work. On the latest 3.6 branch, I get:
```bash
$> osgconv a.obj b.osgb
Error reading file a.obj: not implemented
Warning: failed to load any files```

**Comment by vsg-dev** [comment]
Curious... I'll have a look into it.

**Comment by openscenegraph** [comment]
I have checked in a fix to the OpenSceneGraph-3.6 branch and master to fix this regression:

https://github.com/openscenegraph/OpenSceneGraph/commit/00679c7631fdd588c9ac9831dbf6def0a332099c
